pdt_module/Stixel_msg[] stixels
int32[] ground_corridor